_i18n/de.yml

```
@@ -0,0 +1,90 @@
+meta:
+  description: Entwickle 2D- und 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+header:
+  features: Features
+  showcase: Showcase
+  blog: Blog
+  community: Community
+  assets: Assets
+  download: Download
+  docs: Docs
+  contribute: Mitwirken
+  donate: Spenden
+  language: Sprache
+footer:
+  download: Download
+  documentation: Dokumentation
+  features: Features
+  web_editor: Web Editor
+  release_archive: Release Archiv
+  source_code: Quellcode
+  project: Projekt
+  blog: Blog
+  code_of_conduct: Verhaltenskodex
+  governance: Verwaltung
+  teams: Teams
+  priorities: Prioritäten
+  communities: Communities
+  resources: Ressourcen
+  asset_library: Asset-Bibliothek
+  press_kit: Presse-Kit
+  showcase: Showcase
+  education: Bildung
+  foundation: Stiftung
+  about: Über uns
+  donate: Spenden
+  license: Lizenz
+  privacy_policy: Datenschutzerklärung
+  contact_us: Kontakt
+  text: "Juan Linietsky, Ariel Manzur und <a href='https://github.com/godotengine/godot/blob/master/AUTHORS.md' target='_blank' rel='noopener'>Beitragende</a>. Gehostet von der <a href='https://godot.foundation/' target='_blank' rel='noopener'>Godot Foundation</a>. Website Quellcode <a href='https://github.com/godotengine/godot-website' target='_blank' rel='noopener'>auf  GitHub</a>."
+sponsors:
+  title: "Godot wird gesponsert von:"
+  text: Wenn Sie an einem Unternehmensponsoring interessiert sein, schreiben Sie bitte an <a href='mailto:contact@godot.foundation'>contact@godot.foundation</a>.<br>Spenden als Einzelpersonen können durch den <a href='https://fund.godotengine.org/'>Godot Engine Entwicklungsfonds</a> getätigt werden.
+  cta: Spenden
+home:
+  h1: Deine kostenlose, Open&#8209;Source<br>Game Engine.
+  subtitle: Entwickle 2D- &amp; 3D-Spiele, plattformübergreifenden Projekte oder sogar XR-Ideen!
+  whats_new: Neuigkeiten
+  download_latest: Downloade die neueste Version
```

```suggestion
  download_latest: Lade die neueste Version herunter
```
Can't really conjugate lean words, and we tend to split words like these anyways